dev-infrastructure/templates/mgmt-cluster.bicep

```diff
@@ -246,6 +249,42 @@ module mgmtKeyVault '../modules/keyvault/keyvault.bicep' = {
   }
 }
 
+module cxClusterServiceKeyVaultAccess '../modules/keyvault/keyvault-secret-access.bicep' = [
+  for role in [
+    'Key Vault Secrets User'
+    'Key Vault Certificate User'
+    'Key Vault Certificates Officer'
+  ]: {
```

Comment:
These three roles looks about right for me.
Anything else we need to remove / add @miguelsorianod ?

Reply:
i guess you will need `Key Vault Secrets Officer` as well to be able to create secrets, for things like the the ACR pull secret token etc

Reply:
You are right. We need RW and I thought Key Vault Secrets User gives us that by looking at the comment in https://github.com/Azure/ARO-HCP/blob/3932d27cd29f7207bb277468507b276df1aa2055/dev-infrastructure/modules/keyvault/keyvault-secret-access.bicep#L20 but it seems like the comment wrong it should be `Read secret contents including secret portion of a certificate with private key. Only works for key vaults that use the 'Azure role-based access control' permission model.` Non?

Reply:
changed